spec/system/page_objects/components/new_topic_list_toggle_tab.rb

```diff
@@ -30,7 +30,8 @@ def has_count?(count)
               I18n.t("js.filters.new.#{@name}")
             end
           )
-        find(@selector).text == expected_label
```

Comment:
@OsamaSayegh If the `@selector` is present, this assertion will return immediately which is why you needed the `try_until_success`. Instead what you want is to get Capybara to find the `@selector` with the right text such that it will keep retrying until it finds it or until the timeout is exhaused.